doc/src/manual/faq.md

```diff
@@ -593,6 +593,12 @@ julia> remotecall_fetch(anon_bar, 2)
 1
 ```
 
+### Why does Julia use "*" for string concatenation? Why not "+" or something else?
+
+The [main argument](@ref man-concatenation) against `+` is that string concatenation is not commutative, while `+` is generally used as a commutative operator. While the Julia community recognizes that other languages use different operators and `*` may be unfamiliar for some users, syntax choices always have a subjective element and further discussions about this operator have not turned out to be productive, so it is not likely to change in the short run.
```

Comment:
The nature of the discussion around the operator is not really relevant here. This reads a bit passive-aggressive to me.

Reply:
Sorry about that, English is not my first language. The intent was to convey the idea that the question has been given due consideration.

Reply:
I wish something like this had been merged into the docs - my colleague and I were wondering why such an awkward operator was being used for string concatenation, and at least this provides the rationale. I mean, it still doesn't really make sense to me since `*` is generally used for multiplication which is also a commutative operation (between scalars).

In any case, I think the first sentence of that explanation is great and would make a valuable addition to the docs i.e.

> ### Why does Julia use "*" for string concatenation? Why not "+" or something else?
>
> The [main argument](@ref man-concatenation) against `+` is that string concatenation is not commutative, while `+` is generally used as a commutative operator.

Reply:
Maybe I can reword this. In the past year or so, this issue/discussion came up surprisingly rarely, so the FAQ may not need to address that part.